python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -705,6 +706,15 @@ def hybrid_forward(self, *args, **kwargs):
         raise NotImplementedError
 
 
+class HybridSequentialRNNCell(SequentialRNNCell, HybridBlock):
+    """Sequentially stacking multiple HybridRNN cells."""
+    def __init__(self, prefix=None, params=None):
+        super(HybridSequentialRNNCell, self).__init__(prefix=prefix, params=params)
+
+    def hybrid_forward(self, F, x, *args, **kwargs):
+        raise NotImplementedError
```

Comment:
HybridSequentialRNNCell should have hybrid_forward implemented similarly to SequentialRNNCell.

Reply:
Sorry for the confusion. The hybrid_forward method should have the forward logic similar to what SequentialRNNCell has in its `__call__`

Reply:
@piiswrong do you remember why SequentialRNNCell has `hybrid_forward`?